File: sdk/client/javascript/openapi-test/test/klaytnDebug/others/DebugChaindbProperty.test.js
const OpenSdk = require("opensdk-javascript");
const { expect } = require("@jest/globals");
const { RPC } = require("../../constant");

const sdk = new OpenSdk(new OpenSdk.ApiClient(RPC));

describe('debug_chaindbProperty API', () => {
    test('should return debug_chaindbProperty', (done) => {

        let callbackOne = function (error, data, response) {
            expect(error).toBeNull();
            expect(data.jsonrpc).toBe("2.0");
            expect(data.result).toBeDefined()
            done();
        };

        const property = "0xe17d821e9a8a8736b9aea8c2de1f3a4934ac0a2f";

        sdk.debug.chaindbProperty(property, {}, callbackOne);
    });
});

File: sdk/client/java/openapi-test/src/test/java/opensdk/sdk/apis/debug/others/DebugChaindbPropertyTest.java
package opensdk.sdk.apis.debug.others;

import opensdk.sdk.apis.constant.UrlConstants;
import opensdk.sdk.models.DebugChaindbPropertyResponse;
import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;
import org.klaytn.OpenSDK;

import java.io.IOException;

@DisplayName("Debug RPC Test")
public class DebugChaindbPropertyTest {
    private final OpenSDK sdk = new OpenSDK(UrlConstants.SERVER_URL);

    @Test
    @DisplayName("RPC debug_chaindbProperty")
    void whenRequestValid_ThenCall200ResponseReturns() throws IOException {
        String property = "0xe17d821e9a8a8736b9aea8c2de1f3a4934ac0a2f";

        DebugChaindbPropertyResponse response = sdk.debug.chaindbProperty(property).send();
        response.getResult();
    }
}
